server/src/test/scala/controllers/DropInsControllerSpec.scala

```diff
@@ -0,0 +1,36 @@
+package controllers
+
+import email.GovNotifyEmail
+import module.DRTModule
+import org.scalatestplus.mockito.MockitoSugar
+import org.scalatestplus.play.PlaySpec
+import play.api.test.Helpers._
+import play.api.test._
+
+class DropInsControllerSpec extends PlaySpec with MockitoSugar {
+
+  "DropInsController" should {
+
+    "get published drop-ins" in {
```

Comment:
Nice! Can we add tests for the other 2 end points in this controller too whilst we're here?